tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts

```diff
@@ -13,14 +13,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-import {createSelector, createFeatureSelector} from '@ngrx/store';
+import {createSelector, createFeatureSelector, select} from '@ngrx/store';
```

Comment:
I do not see corresponding test changes. Especially since there are non-trivial logic in the selector, I will strongly recommend writing the tests.